src/main.cpp

```diff
@@ -768,6 +768,11 @@ static int main0(int argc, char **argv) {
                     return EXIT_FAILURE;
                 }
                 dynamic_linker = buf_ptr(linker_args.at(i));
+            } else if (buf_eql_str(arg, "-E") ||
```

Comment:
Why `-E`? for a C compiler it means "preprocessor only". I'm hesitant to overload that meaning.

Reply:
`-E` is an alias for `--export-dynamic` in [`ld`](https://linux.die.net/man/1/ld)/[`lld`](https://github.com/llvm-mirror/lld/blob/64b024a57c56c3528d6be3d14be5e3da42614a6f/ELF/Options.td#L440). Note that this check is only for linker args (i.e. things passed to `zig cc` via `-Wl`).

Reply:
To be more clear, since I realized I didn't include this in the OP:

This is meant to fix `warning: unsupported linker arg: --export-dynamic` when doing `zig cc -Wl,--export-dynamic` or `zig cc -Wl,-E` (or other ways of passing linker args, but those [aren't supported by `zig cc` yet](https://github.com/ziglang/zig/issues/4784)). Full context is [here](https://github.com/luvit/luvi/issues/231#issuecomment-604227535)